package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "generator-eslint",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
Please don't change the version. This is done automatically as part of the release process.